eng/scripts/Test-Code.ps1

```diff
@@ -11,7 +11,8 @@ param(
     [switch] $CollectCoverage,
     [switch] $OpenReport,
     [switch] $TestNativeBuild,
-    [switch] $OnlyBuild
+    [switch] $OnlyBuild,
+    [string] $ScopingBuildInfoPath = $null
```

Comment:
I would just call it BuildInfoPath to be consistent with all the other scripts that handle build_info.json

Reply:
I started here, but given that I'm only using this for scoping I'm going to keep this change.